ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalDiocesanConsent.cshtml.cs

```diff
@@ -9,30 +10,29 @@ namespace ApplyToBecomeInternal.Pages.TaskList.LegalRequirements
 {
 	public class LegalDiocesanConsentModel : LegalModelBase
 	{
-		public LegalDiocesanConsentModel(ILegalRequirementsRepository legalRequirementsRepository,
-			IAcademyConversionProjectRepository academyConversionProjectRepository) :
-			base(legalRequirementsRepository, academyConversionProjectRepository)
+		public LegalDiocesanConsentModel(IAcademyConversionProjectRepository academyConversionProjectRepository) :
+			base(academyConversionProjectRepository)
 		{
 		}
 
 		[BindProperty] public string Approved { get; set; }
 
 		public void OnGet(int id)
 		{
-			Approved = LegalRequirements.DiocesanConsent.ToString();
+			Approved = Requirements.DiocesanConsent.ToString();
 		}
 
 		public async Task<IActionResult> OnPostAsync(int id)
 		{
-			LegalRequirements.DiocesanConsent = Approved switch
+			Requirements.DiocesanConsent = Approved switch
```

Comment:
I know you didn't make this change. But there is a ToDescription extension method that can be used instead of this switch statement. Would save some screen space.